### PR TITLE
Enable God mode to wildcard admin institute #4477

### LINKED ISSUE #4477: Enable God mode to wildcard admin institute
This is _the last_ thing that the current God mode is unable to regenerate correctly, in the list of items that need proper treatment.
This needs wildcard because the institute varies based on the admin account, which varies from developer to developer and cannot be forced.
Wildcarding the institute name only does not work because the institute name can be empty, and empty string right after an HTML tag is unfortunately one of wildcard's limitation.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #4477 
Tested in the admin UI tests, regeneration works perfectly.

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/bodyFooter.tag [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 <%@ tag import="teammates.common.util.Config" %>
+<%@ attribute name="isAdmin" %>
 <div id="footerComponent" class="container-fluid">
     <div class="container">
         <div class="row">
             <div class="col-md-2">
                 <span>[<a href="/index.html">TEAMMATES</a> V<%= Config.inst().getAppVersion() %>]</span>
             </div>
-            <div class="col-md-8">
+            <div class="col-md-8" <c:if test="${isAdmin}">id="admin-institute"</c:if>>
```
Comment (thyageshm) [inline src/main/webapp/WEB-INF/tags/bodyFooter.tag]: just one minor suggestion here. The standard is to use camel casing for ids and small letters with dashes for classes. See if we can change this to follow them

**Comment by thyageshm** [comment]
just a minor suggestion. Also, try to delete the html files and generate them from scratch to verify thoroughly.

File: src/main/webapp/WEB-INF/tags/bodyFooter.tag (hunk 1 above)
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/bodyFooter.tag]: That's perfectly possible, but I dunno we have such convention o.O that means the style guide is severely outdated.

**Comment by wkurniawan07** [comment]
Turns out there are two HTMLs affected, both from #4425. I bet it's because they're not generated with God mode.
On a related note, I don't use the delete-everything-then-regenerate workflow. It's dangerous since what's regenerated can be wrong. While running the full test again will catch those errors, it means you have to run the full test twice to regenerate. Talk about efficiency.

**Comment by thyageshm** [comment]
Now that you mention it, I think the "Id" standard doesn't seem to be followed in a steadfast manner. I saw 3 different ways in which "Id"s were defined in instructorCommentsPageEditFrc.html file. I think we should handle that as well in another issue..

Also, when I delete and run it, I check git diff to wnsure only the lines I expected have changed :-P 
And, I dont think we delete ALL the files, so it should be pretty fast... Anyway, I suggested that method cos wildcards in admin pages often dont let Godmode regenerate the files since the tests pass anyway. Unless you forced it to regenerate or modified the html file deliberately (which is what I do when it comes to Admin pages), the confirmation is not complete.

Ok, just ensure that the tests pass (dev green) and we can fix those files in #4447

**Pull request merged**